sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusTriggerMetrics.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.Azure.WebJobs.ServiceBus.Listeners
 {
-    internal class ServiceBusTriggerMetrics : ScaleMetrics
+    public class ServiceBusTriggerMetrics : ScaleMetrics
```

Comment:
why does this need to be public. shouldn't have to be

Reply:
@chiangvincent asked `ServiceBusScaleMonitor` to be paublic for SC redesign. ServiceBusScaleMonitor inherits from  
`public class ServiceBusScaleMonitor : IScaleMonitor<ServiceBusTriggerMetrics>`